src/test/src/routers/soleTraderCorrespondanceAddressManualRouter.test.ts

```diff
@@ -2,169 +2,169 @@ import supertest from "supertest";
 import app from "../../../main/app";
 const router = supertest(app);
 
-describe("GET /sole-trader/correspondance-address-manual", () => {
```

Comment:
So I changed it from address-correspondance-manual to correspondance-address-manual. 
As that is the name of all our file as well. And its known as correspondance address instead of address correspondance.

Reply:
So thats just the url, you can just have that in routes/index.js.

Reply:
addressed the comments